.vscode/settings.json

```diff
@@ -13,5 +13,6 @@
     "files.associations": {
         "*.flow": "plaintext",
     },
-    "flow.enabled": false
-}
\ No newline at end of file
+    "flow.enabled": false,
+    "jest.jestCommandLine": "yarn jest"
```

Comment:
I caught this and the next file during my local test - it was running `yarn test` which caused a bunch of unrelated stuff to run (and the results failed to appear in VS Code). So this allows the tests to now run correctly!